web/quiz_views.py

```
@@ -16,8 +16,167 @@
     QuizOptionFormSet,
     QuizQuestionForm,
     TakeQuizForm,
+    QuizQuestionSpecializedForm,
 )
-from .models import Quiz, QuizQuestion, UserQuiz
+from .models import Quiz, QuizQuestion, UserQuiz, Course, Session
+
+
+@login_required
+def create_course_exam(request, course_id=None, session_id=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_course_exam`

(ANN201)